src/components/SelectionList/BaseListItem.js

```diff
@@ -31,6 +31,20 @@ function BaseListItem({
     const isUserItem = lodashGet(item, 'icons.length', 0) > 0;
     const ListItem = isUserItem ? UserListItem : RadioListItem;
 
+    const textStyle = StyleUtils.combineStyles(
```

Comment:
I don't think we need `StyleUtils.combineStyles`. Just return an array here

Reply:
done

Reply:
When we have a combination of arrays of objects and objects that needs to be flatten out. Here we only have objects.